src/main/java/com/nparo/songr/RouteController.java

```diff
@@ -0,0 +1,52 @@
+package com.nparo.songr;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestParam;
+
+@Controller
+public class RouteController {
+//  Route getters:
+  @GetMapping("/")
+  public String getRoot() {
+    return "index";
+  }
+
+  @GetMapping("/hello")
+  public String getHello() {
+    return "hello";
+  }
+
+  @GetMapping("/capitalize/{words}")
+  public String getCapitalize(@PathVariable String words, Model m) {
+    m.addAttribute("words", setCapitalize(words));
+    return "capitalize";
+  }
+
+  @GetMapping("/reverse")
+  public String getReverse(@RequestParam(required = false, defaultValue = "Reversed Sentence Goes Here") String sentence,
+                           Model m) {
+    String reverse = setReverse(sentence);
+    m.addAttribute("sentence", reverse);
+    return "reverse";
+  }
+
+//  Helper Methods:
+  public static String setCapitalize(String words) {
```

Comment:
I'm not sure it's worth the effort to create a method just to call `.toUpperCase()`.